src/wwwroot/partials/settings/my-plan.html

```diff
@@ -7,21 +7,21 @@
   </div>
   <div class="section--wrapper my-plan--info-container">
     <div class="item flex flex--wrap first">
-      <div class="flex one flex--wrap">
+      <div class="flex one flex--wrap current-plan-price">
```

Comment:
mmm...ok. I think that we can improve this using [Data Attributes](https://www.w3schools.com/tags/att_global_data.asp) e.g. `<div class="flex one flex--wrap" data-test_identifier = "current-plan-price">`
I mean, I think that we have to avoid use css classes only for identify an element for UI test.  Html5 has this attribute so that we can store custom data like this.

What do you think @andresmoschini , @mvillaseco ?